src/github-release.ts

```diff
@@ -78,6 +78,14 @@ export class GitHubRelease {
   }
 
   async createRelease(): Promise<ReleaseResponse | undefined> {
+    // Attempt to lookup the package name from a well known location, such
+    // as package.json, if none is provided:
+    if (!this.packageName && this.releaseType) {
```

Comment:
I've just been specifying package-name in my monorepo implementations, this seems smarter.